docs/docsite/rst/network/user_guide/faq.rst

```diff
@@ -0,0 +1,34 @@
+.. _network_faq:
+
+*******************
+Ansible Network FAQ
+*******************
+
+How can I improve performance for network playbooks?
+====================================================
+
+* Set ``strategy: free`` at the playbook level if you are running on multiple hosts
+
```

Comment:
Maybe something like:

*What* is this option
*When* would this make sense (i.e. how do you know if this is something to consider)